shared/people/todo/index.tsx

```diff
@@ -16,27 +16,25 @@ export type Props = {
 }
 
 export const Task = (props: Props) => (
-  <PeopleItem format="multi" badged={props.badged} icon={<Icon type={props.icon} />}>
-    <Text type="Body" style={styles.instructions}>
+  <PeopleItem format="multi" badged={props.badged} icon={<Kb.Icon type={props.icon} />}>
+    <Kb.Text type="Body" style={styles.instructions}>
       {props.instructions}
-    </Text>
-    <Box style={styles.actionContainer}>
+    </Kb.Text>
+    <Kb.Box style={styles.actionContainer}>
       {props.showSearchBar ? (
         <PeopleSearch style={styles.search} />
       ) : (
-        <Button
+        <Kb.Button
           small={true}
           label={props.confirmLabel}
           onClick={props.onConfirm}
           style={{marginRight: Styles.globalMargins.small}}
         />
       )}
       {props.dismissable && (
-        <Text type="BodyPrimaryLink" onClick={props.onDismiss}>
-          Later
-        </Text>
+        <Kb.Button small={true} label="Follow later" mode="Secondary" onClick={props.onDismiss} />
```

Comment:
did this change every todo's dismiss label to "Follow later" ?

Reply:
Hmm that's a good point. Also, WDYT about it being a button instead of a label @cecileboucheron?

Reply:
I'll make a fix to the follow text at least